web/src/ui_init.js

```diff
@@ -104,7 +105,6 @@ import * as stream_settings_ui from "./stream_settings_ui";
 import * as sub_store from "./sub_store";
 import * as timerender from "./timerender";
 import * as tippyjs from "./tippyjs";
-import * as top_left_corner from "./top_left_corner";
```

Comment:
This commit is almost certainly broken, since I don't see the new module being imported. Also looking at `git grep top_left_*`, I think we have a bunch of other things to rename in follow-up commits -- next time I'd do this sort of thing as a prep PR that just does the rename -- a lot easier to quickly review/merge that and it'll also catch mistakes like this for you.

In general these

Reply:
Actually, it seems you did this correctly and I somehow misread; in any case, a good reason to do big renames in their own prep PRs is that it makes it easier to review/verify :).